io.edgehog.devicemanager.LedBehavior.json

```diff
@@ -0,0 +1,15 @@
+{
+  "interface_name": "io.edgehog.devicemanager.LedBehavior",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "properties",
```

Comment:
I would use a datastream here, since I would use them with a fire-and-forget logic